src/main/java/uk/gov/companieshouse/api/accounts/utility/BindingResultErrorToErrorMapper.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.companieshouse.api.accounts.utility;
+
+import org.springframework.stereotype.Component;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+import uk.gov.companieshouse.api.accounts.model.validation.Error;
+import uk.gov.companieshouse.api.accounts.validation.ErrorMessageKeys;
+import uk.gov.companieshouse.api.accounts.validation.ErrorType;
+import uk.gov.companieshouse.api.accounts.validation.LocationType;
+
+import java.util.ArrayList;
+
+
+@Component
+public class BindingResultErrorToErrorMapper {
+
+    String location;
```

Comment:
This is a singleton component and cannot have state, this location should only be created locally.